pkgs/applications/editors/emacs/make-emacs.nix

```diff
@@ -339,6 +343,7 @@ mkDerivation (finalAttrs: {
   ++ lib.optional withXinput2 "--with-xinput2"
   ++ lib.optional withXwidgets "--with-xwidgets"
   ++ lib.optional withSmallJaDic "--with-small-ja-dic"
+  ++ lib.optional (!withCompressInstall) "--without-compress-install"
```

Comment:
This reads incredibly ugly, we should port emacs to using `enableFeature`.

Reply:
Indeed! Something like `lib.withFeature withCompressInstall "compress-install"` would read much better